_gsocproposals/2020/proposal_TMVADecisionTreeInference.md

```diff
@@ -0,0 +1,38 @@
+---
+title: TMVA Deep Learning Developments - Decision Tree Inference, Expansion of Supported Format
+layout: gsoc_proposal
+project: TMVA
+year: 2020
+organization:
+  - CERN
+---
+
+# Description
+
+Toolkit for Multivariate Analysis [(TMVA)](http://root.cern/tmva) is a multi-purpose machine learning toolkit integrated into the [ROOT](http://root.cern) scientific software framework, used in many particle physics data analysis and applications. We provide in TMVA the functionality for building deep neural networks including fully connected, convolutional and recurrent layers.
+
+We have already a very efficient inference engine for decision tree based ML algorithms developed in TMVA. However, currently this engine only supports xgBoost model files. We would like to expand this support to existing TMVA XML formats and Scikit-learn pickles, as well as to support serialization of the model in ROOT format and conversion of other models into ROOT format.
```

Comment:
You might wish to add a link to the `xgBoost` model…

Reply:
Done